frontend/app/components/HeaderAppBar.tsx

```diff
@@ -40,6 +45,10 @@ export default function HeaderAppBar (props: HeaderAppBarProps) {
             </Grid>
           </Grid>
         </Grid>
+        <Grid item xs='auto' container justifyContent='space-around'>
+          <Button color='inherit' target='_blank' href={props.helpURL}>Help</Button>
```

Comment:
Both buttons are with inherit color, with no contrast with background color. May be an issue with accessibility?

Reply:
I'm not sure the buttons have to be distinguishable from the app bar if they're just links (the button text does of course). I was kinda following this example here: https://mui.com/material-ui/react-app-bar/#responsive-app-bar-with-drawer If the buttons are focused, they are somewhat distinguishable.

What are you suggesting in terms of a change here?

Reply:
I see. I have no issue then based on the MUI web page.